crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__all.snap

```diff
@@ -0,0 +1,174 @@
+
+error[Compatibility E02001]: variant mismatch
+   ┌─ /fixtures/upgrade_errors/all_v2/sources/UpgradeErrors.move:64:9
+   │
+60 │     public enum EnumNewVariant {
+   │                 -------------- Enum definition
+   ·
+64 │         D // new variant
+   │         ^ New unexpected variant 'D'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's variants for enum 'EnumNewVariant' including the ordering.
+
+error[Compatibility E02001]: variant mismatch
+   ┌─ /fixtures/upgrade_errors/all_v2/sources/UpgradeErrors.move:68:17
+   │
+68 │     public enum EnumVariantMissing {
+   │                 ^^^^^^^^^^^^^^^^^^ Missing variant 'B'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's variant 'B' for enum 'EnumVariantMissing' including the ordering.
+
+error[Compatibility E01001]: missing public declaration
+   ┌─ /fixtures/upgrade_errors/all_v2/sources/UpgradeErrors.move:74:9
+   │
+74 │     fun function_to_have_public_removed() {}
+   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Function 'function_to_have_public_removed' has lost its public visibility
+   │
+   = Functions are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original function's 'public' visibility for function 'function_to_have_public_removed'.
+
+error[Compatibility E03001]: function signature mismatch
+   ┌─ /fixtures/upgrade_errors/all_v2/sources/UpgradeErrors.move:82:16
+   │
+82 │     public fun function_add_arg(a: u64) {}
+   │                ^^^^^^^^^^^^^^^^ Expected 0 parameters, found 1
+   │
+   = Functions are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original function's parameters for function 'function_add_arg', expected 0 parameters.
+
+error[Compatibility E03001]: function signature mismatch
+   ┌─ /fixtures/upgrade_errors/all_v2/sources/UpgradeErrors.move:85:16
+   │
+85 │     public fun function_remove_arg() {}
+   │                ^^^^^^^^^^^^^^^^^^^ Expected 1 parameter, found 0
+   │
+   = Functions are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original function's parameter for function 'function_remove_arg', expected 1 parameter.
+
+error[Compatibility E03001]: function signature mismatch
+   ┌─ /fixtures/upgrade_errors/all_v2/sources/UpgradeErrors.move:88:36
+   │
+88 │     public fun function_change_arg(a: u8) {} // now has u8 instead of u64
+   │                                    ^ Unexpected parameter u8, expected u64
+   │
+   = Functions are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original function's parameter for function 'function_change_arg'.
+
+
+Upgrade failed, this package requires changes to be compatible with the existing package. Its upgrade policy is set to 'compatible'.
```

Comment:
We should move this message to the top, it seems like it would be easy to lose it, but it's the "TL;DR" of all the other stuff here.

Reply:
Spoke with @jordanjennings-mysten -- he explained the rationale of putting it last, which is that it will effectively be the first thing that people see, because it's the last thing output before the terminal prompt, so all good, feel free to disregard this comment!

Reply:
I think where we landed here was that we should leave the ordering as is to keep formatting in-line with the other move errors, so I'll mark it resolved.